packages/@biomejs/biome/configuration_schema.json

```diff
@@ -4132,7 +4132,6 @@
 			"properties": {
 				"attributes": {
 					"description": "Additional attributes that will be sorted.",
-					"default": ["class", "className"],
```

Comment:
Is it expected?

Reply:
Yes. This avoids some allocations and doesn't change the behavior of the rule.